src/containers/shared/components/TransactionTable/TransactionTableRow.tsx

```diff
@@ -61,3 +68,7 @@ export const TransactionTableRow = ({ tx }: Props) => {
     </li>
   )
 }
+
+TransactionTableRow.defaultProps = {
```

Comment:
If you export `Props` this isn't needed.

Reply:
You mean export the interface Props? Why would that remove the need for the default prop?

Reply:
I don't know why but it makes eslint stop complaining about it.

Reply:
The purpose of the default prop here is actually to guarantee the default value of the field is false. Are you saying that even if this is removed the default value will still be false?

Reply:
You can set the default in the parameters